README.md

```diff
@@ -65,17 +66,21 @@ Prerequisites:
 
 Once you have completed all the pre-requisites building Realm is done with a simple command
 
-    ./gradlew androidJar
+    ./gradlew assembleExamples
```

Comment:
No, the distribution package has it own target.

The `gradle-plugin` project has no dependencies
The `realm` project has no dependencies
The `examples` project depends on `gradle-plugin` and `realm`

So running `assembleExamples` is an easy way to build everything you need.
I should probably create an `assemble` task that does nothing apart from depending on `assembleExamples`